common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -67,6 +67,34 @@ public static X509TrustManager getDefaultX509TrustManager() throws KeyManagement
     }
 
     /**
+     * Utility that exposes common TLS constants.
+     */
+    public static final class Constants {
+        private Constants() {}
+
+        /**
+         * Returns the maximum length (in bytes) of plaintext data.
+         */
+        public static int maxPlaintextLength() {
+            return NativeConstants.SSL3_RT_MAX_PLAIN_LENGTH;
```

Comment:
Getting rid of the plaintext length for now, as it won't be immediately needed by Netty.